### PR TITLE
Added Media section permission for anonymous user to be able to visua…

### PR DESCRIPTION
**Pull request by zjabibexa**
A default installation of Ibexa DXP does not include `Media` section in `Anonymous` role permissions.
Following the tutorial we will have this error : 

`
An exception has been thrown during the rendering of a template ("Error when rendering "http://localhost:8000/_fragment?_path=_format%3Dhtml%26_locale%3Den_GB%26_controller%3Dibexa_content%253A%253AviewAction" (Status code is 302).") in "@standard/ui/field_type/preview/ezdam.html.twig".
`
We see clearly a redirection ( 302 ) due to a request made to a protected resource.

If we add the permission, the exception disappears and we see the photo without problems.

| Question      | Answer
| ------------- | ---
| JIRA Ticket   |  N/A 
| Versions      |  4.3+

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

**Comment by DominikaK** [comment]
@zjabibexa, are you testing this on an Ibexa Content installation? Then I think this is related to this bug: https://issues.ibexa.co/browse/IBX-2298

**Comment by zjabibexa** [comment]
Yes exactly, I tested on an Ibexa Content Installation, on Experience Installation I see the roles but on the Content no.

File: docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md [hunk 1]
```diff
@@ -173,6 +173,17 @@ Because this template will be rendered inside a table, it starts with a `<tr>` t
     </td>
 </tr>
 ```
+### Add Media permission
+
+To be able to visualise the `photo` field you have to add `read` permission to `Media` section.
```
Comment (DominikaK) [inline docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md:178]:
```suggestion
To be able to view the `photo` Field you have to add a `read` permission to `Media` section.
```

File: docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md [hunk 2]
```diff
@@ -173,6 +173,17 @@ Because this template will be rendered inside a table, it starts with a `<tr>` t
...
+
+From **Admin** (gear icon) in the left menu, go to the **Roles** management screen and click on the **Anonymous** Role.
```
Comment (DominikaK) [inline docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md:180]:
```suggestion
From **Admin** (gear icon) in the left menu, go to the **Roles** management screen and click the **Anonymous** Role.
```

File: docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md [hunk 3]
```diff
@@ -173,6 +173,17 @@ Because this template will be rendered inside a table, it starts with a `<tr>` t
...
+
+![Policies for the Anonymous Role without Media section](step5_admin_anonymous_policies_without_media_section.png)
+
+Edit the **Content/Read** policy line to add `Media` section to **Limitation** along with the `Standard` section.
```
Comment (DominikaK) [inline docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md:184]:
```suggestion
Edit the **Content/Read** Policy line to add the `Media` section to **Limitation** along with the `Standard` section.
```

**Review by DominikaK — CHANGES_REQUESTED** [review]
Just a few cosmetic fixes to fit out style guide.

**Review by DominikaK — APPROVED** [review]

**Pull request merged**